### ISSUE TITLE
[Enocean] Bridge goes offline after a while

### ISSUE THREAD

**Issue by tokest** [openhab/openhab-addons#15330]
## Expected Behavior
After new install OH 4.0.1 on Nuc i5 with Ubuntu 22.04.2 LTS Ubuntu the EnOcean bridge works normal, and everything looks good. My EnOcean buttons send messages and the EnOcean bridge receives these.   

## Current Behavior
After a few hours suddenly the EnOcean bridge goes offline an I get a configuration error/pending message. The strange thing is, that even there is a config error and the thing (EnOcean Bridge) is not running, because the port was not found, the EnOcean rollershutter is working (at least at the last hour, and it is still working). I’m using EnOcean for the blind buttons (and other stuff) and control the homematic rollershutter.

> tail -f /var/log/openhab/*.log | grep -e nocean
>2023-07-29 16:26:42.918 [INFO ] [openhab.event.ChannelTriggeredEvent ] - enocean:rockerSwitch:gtwy:7a72d69ba6:rockerswitchA triggered DIR2_PRESSED
>2023-07-29 16:26:42.921 [INFO ] [openhab.event.ChannelTriggeredEvent ] - enocean:rockerSwitch:gtwy:05d32df82c:rockerswitchA triggered DIR2_PRESSED
>2023-07-29 16:26:43.094 [INFO ] [openhab.event.ChannelTriggeredEvent ] - enocean:rockerSwitch:gtwy:7a72d69ba6:rockerswitchA triggered DIR2_RELEASED
>2023-07-29 16:26:43.095 [INFO ] [openhab.event.ChannelTriggeredEvent ] - enocean:rockerSwitch:gtwy:05d32df82c:rockerswitchA triggered DIR2_RELEASED
>2023-07-29 16:26:44.851 [INFO ] [openhab.event.ChannelTriggeredEvent ] - enocean:rockerSwitch:gtwy:7a72d69ba6:rockerswitchA triggered DIR2_PRESSED
>2023-07-29 16:26:44.852 [INFO ] [openhab.event.ChannelTriggeredEvent ] - enocean:rockerSwitch:gtwy:05d32df82c:rockerswitchA triggered DIR2_PRESSED
>2023-07-29 16:26:45.027 [INFO ] [openhab.event.ChannelTriggeredEvent ] - enocean:rockerSwitch:gtwy:7a72d69ba6:rockerswitchA triggered DIR2_RELEASED
>2023-07-29 16:26:45.029 [INFO ] [openhab.event.ChannelTriggeredEvent ] - enocean:rockerSwitch:gtwy:05d32df82c:rockerswitchA triggered DIR2_RELEASED
>2023-07-29 16:26:45.926 [INFO ] [openhab.event.ChannelTriggeredEvent ] - enocean:rockerSwitch:gtwy:7a72d69ba6:rockerswitchA triggered DIR1_PRESSED
>2023-07-29 16:26:45.929 [INFO ] [openhab.event.ChannelTriggeredEvent ] - enocean:rockerSwitch:gtwy:05d32df82c:rockerswitchA triggered DIR1_PRESSED
>2023-07-29 16:26:46.084 [INFO ] [openhab.event.ChannelTriggeredEvent ] - enocean:rockerSwitch:gtwy:7a72d69ba6:rockerswitchA triggered DIR1_RELEASED
>2023-07-29 16:26:46.086 [INFO ] [openhab.event.ChannelTriggeredEvent ] - enocean:rockerSwitch:gtwy:05d32df82c:rockerswitchA triggered DIR1_RELEASED
>2023-07-29 16:27:01.679 [INFO ] [ab.event.ThingStatusInfoChangedEvent] - Thing 'enocean:bridge:f424041d7c' changed from OFFLINE (CONFIGURATION_ERROR): Port could not be found to OFFLINE (CONFIGURATION_PENDING): opening serial port...
>2023-07-29 16:27:01.687 [INFO ] [ab.event.ThingStatusInfoChangedEvent] - Thing 'enocean:bridge:f424041d7c' changed from OFFLINE (CONFIGURATION_PENDING): opening serial port... to OFFLINE (CONFIGURATION_ERROR): Port could not be found
>2023-07-29 16:27:05.912 [INFO ] [openhab.event.ChannelTriggeredEvent ] - enocean:rockerSwitch:gtwy:7a72d69ba6:rockerswitchA triggered DIR2_PRESSED
>2023-07-29 16:27:05.912 [INFO ] [openhab.event.ChannelTriggeredEvent ] - enocean:rockerSwitch:gtwy:05d32df82c:rockerswitchA triggered DIR2_PRESSED
>2023-07-29 16:27:06.040 [INFO ] [openhab.event.ChannelTriggeredEvent ] - enocean:rockerSwitch:gtwy:7a72d69ba6:rockerswitchA triggered DIR2_RELEASED
>2023-07-29 16:27:06.040 [INFO ] [openhab.event.ChannelTriggeredEvent ] - enocean:rockerSwitch:gtwy:05d32df82c:rockerswitchA triggered DIR2_RELEASED
>2023-07-29 16:27:07.413 [INFO ] [openhab.event.ChannelTriggeredEvent ] - enocean:rockerSwitch:gtwy:7a72d69ba6:rockerswitchA triggered DIR2_PRESSED
>2023-07-29 16:27:07.416 [INFO ] [openhab.event.ChannelTriggeredEvent ] - enocean:rockerSwitch:gtwy:05d32df82c:rockerswitchA triggered DIR2_PRESSED
>2023-07-29 16:27:07.542 [INFO ] [openhab.event.ChannelTriggeredEvent ] - enocean:rockerSwitch:gtwy:7a72d69ba6:rockerswitchA triggered DIR2_RELEASED
>2023-07-29 16:27:07.542 [INFO ] [openhab.event.ChannelTriggeredEvent ] - enocean:rockerSwitch:gtwy:05d32df82c:rockerswitchA triggered DIR2_RELEASED
>2023-07-29 16:27:08.773 [INFO ] [openhab.event.ChannelTriggeredEvent ] - enocean:rockerSwitch:gtwy:7a72d69ba6:rockerswitchA triggered DIR1_PRESSED
>2023-07-29 16:27:08.774 [INFO ] [openhab.event.ChannelTriggeredEvent ] - enocean:rockerSwitch:gtwy:05d32df82c:rockerswitchA triggered DIR1_PRESSED
>2023-07-29 16:27:08.792 [INFO ] [ab.event.ThingStatusInfoChangedEvent] - Thing 'enocean:bridge:f424041d7c' changed from OFFLINE (CONFIGURATION_ERROR): Port could not be found to OFFLINE (CONFIGURATION_PENDING): opening serial port...
>2023-07-29 16:27:08.799 [INFO ] [ab.event.ThingStatusInfoChangedEvent] - Thing 'enocean:bridge:f424041d7c' changed from OFFLINE (CONFIGURATION_PENDING): opening serial port... to OFFLINE (CONFIGURATION_ERROR): Port could not be found
>2023-07-29 16:27:08.934 [INFO ] [openhab.event.ChannelTriggeredEvent ] - enocean:rockerSwitch:gtwy:7a72d69ba6:rockerswitchA triggered DIR1_RELEASED
>2023-07-29 16:27:08.935 [INFO ] [openhab.event.ChannelTriggeredEvent ] - enocean:rockerSwitch:gtwy:05d32df82c:rockerswitchA triggered DIR1_RELEASED

## Steps to Reproduce (for Bugs)

1. Install Openhab 4.0.1
2. Install EnOcean Addon 
3. Install EnOcean USB300
4. Wait about 2-4 hours (its not clear how long to wait)
5. Issue appears iff EnOcean bridge goes in state error: config or (CONFIGURATION_ERROR) or (CONFIGURATION_PENDING) appears in the openhab logfile.

## Your Environment

* Version used: (e.g., openHAB and add-on versions)
* Environment name and version (e.g. Chrome 76, Java 8, Node.js 12.9, ...):
* Operating System and version (desktop or mobile, Windows 10, Raspbian Buster, ...):
Hardware: Nuc i5
Ubuntu: 22.04.2 LTS
Openhab: 4.0.1
Addon: OnOcean 4.0.1

**Comment by tokest** [comment]
I removed the openhab installation and installed it again with the same version 4.0.1-1. Than I restored the configuration again with the restore script. The behavior is not exactly the same, but maybe there are some helpfull information in the logfile. The difference to my case before is, that the bridge was never seen in the stat "online" in the gui. 
[OH startup logfile.log](https://github.com/openhab/openhab-addons/files/12207487/pcn1-OH-4-0-1_EnOceanUSB300-log.-.Kopie.log)
